pkg/reconciler/eventlistener/resources/deployment.go

```diff
@@ -37,11 +37,22 @@ const (
 )
 
 var (
-	strongerSecurityPolicy = corev1.PodSecurityContext{
+	baseStrongerSecurityPolicy = corev1.PodSecurityContext{
 		RunAsNonRoot: ptr.Bool(true),
+		SeccompProfile: &corev1.SeccompProfile{
+			Type: corev1.SeccompProfileTypeRuntimeDefault,
+		},
 	}
 )
 
+func getStrongerSecurityPolicy(cfg *config.Config) corev1.PodSecurityContext {
+	securityContext := baseStrongerSecurityPolicy
+	securityContext.RunAsUser = ptr.Int64(cfg.Defaults.DefaultRunAsUser)
+	securityContext.RunAsGroup = ptr.Int64(cfg.Defaults.DefaultRunAsGroup)
+	securityContext.FSGroup = ptr.Int64(cfg.Defaults.DefaultRunAsGroup)
+	return securityContext
+}
+
```

Comment:
We already setting these values here https://github.com/tektoncd/triggers/blob/main/pkg/reconciler/eventlistener/resources/container.go#L53-L68

except FSGroup 

if you want to add FSGroup can you update this https://github.com/tektoncd/triggers/blob/main/pkg/reconciler/eventlistener/resources/container.go#L53-L68 part of the code

Reply:
I checked my specific use case again(istio injected sidecars) and it appears runAsGroup and runAsUser is set for that container. 

However, i would like to keep seccompProfile as it is not set and therefore violates restricted pod security standards. 
FSGroup is not available in container securityContext, only PodSecurityContext. 

I can remove runAsGroup and runAsUser if you do not wish to have a pod level default.

Reply:
I think we can keep for pod level as well i don't see issue

cc @khrm @dibyom

Reply:
Is there anything that is missing from this PR now or changes that needs to be done?

Reply:
Ping @savitaashture @khrm